specification/components/schemas/eRS-CreateFromCommunicationRequestActionLater-Request-Parameters.yaml

```diff
@@ -0,0 +1,290 @@
+title: eRS-CreateFromCommunicationRequestActionLater-Request-Parameters
+type: object
+required:
+  - resourceType
+  - meta
+  - parameter
+properties:
+  resourceType:
+    type: string
+    enum:
+      - 'Parameters'
+    example: 'Parameters'
+  meta:
+    type: object
+    required:
+      - profile
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CreateFromCommunicationRequestActionLater-Request-Parameters-1"
+          example: "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-CreateFromCommunicationRequestActionLater-Request-Parameters-1"
+        minItems: 1
+        maxItems: 1
+  parameter:
+    type: array
+    description: |
+      | Field                             | Cardinality | Notes                                      |
+      | --------------------------------- | ----------- | ------------------------------------------ |
+      | `createFrom`                      | `1..1`      |                                            |
+      | `guidance`                        | `1..1`      |                                            |
+      | `newReferralPriority`             | `1..1`      |                                            | 
+      | `reviewComments`                  | `0..1`      |                                            |
+      | `guidanceAttachmentFile`          | `0..*`      |                                            | 
```

Comment:
I think it would be good to indicate the difference between the `guidanceAttachmentFile` and `reviewAttachmentFile` properties here if we can, perhaps via the `description` field for each property. Maybe something like "Property defining files to be included with the supplied `guidance` text" for `guidanceAttachmentFile` and "Property defining files to be associated with the created `ReferralRequest`" for `reviewAttachmentFile`.